gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/ProductDependencyIntrospectionPlugin.java

```diff
@@ -103,8 +101,13 @@ private static String getMinimumProductVersion(Project project, String group, St
             return Optional.empty();
         }
 
-        return Optional.of(ProductDependencyLockFile.fromString(
-                GFileUtils.readFile(lockFile), project.getVersion().toString()));
+        try {
+            String rawLockFile = new String(Files.readAllBytes(lockFile.toPath()), StandardCharsets.UTF_8);
```

Comment:
surprised intellij didn't suggest `Files#readString`

Reply:
Unfortunately, we are still on source compat 1.8 and I did not want to increase it as part of this pr :(

Currently checking if this is still required or if we can raise it to 11.

Reply:
We are good to use source compat 11 now!